scripts/readme.md

```diff
@@ -29,13 +29,19 @@ release.
 4. Commit the changes - use the commit message '<emoji> v<VERSION>'. e.g. `git commit -m '🤦‍ v2.7.0'` (note the emoji is 
 used literally, as opposed to ':facepalm:').
 5. Run `npm run release`, which will push the commit/tag to GitHub and publish to NPM.
-6. Proceed to the [Follow-Up section](#follow-up) of this document to run manual follow-up tasks.
+6. Proceed to the [Follow-Up section](#follow-up-steps) of this document to run manual follow-up tasks.
 
 # Follow-Up Steps
 
 The following steps should be always run, regardless of whether an automated or manual release was performed.
 
 1. Publish the release notes in GitHub using GitHub's [release notes form](https://github.com/ionic-team/stencil/releases/new).
+   1. If anyone from the community contributed commit(s) to this release, append the following to the end of the GitHub release notes:
+```md
+## Thanks
+
+🎉 Thanks <GitHub_Usernames> for their contributions! 🎉
```

Comment:
would it make sense for us to put PR numbers in there too?

Reply:
I suppose it's up to us as a team.

https://github.com/ionic-team/stencil/releases/tag/v4.0.2 has an example of us recently doing this.

TBH I haven't included it since that requires more manual work than scanning https://github.com/ionic-team/stencil/commits/main and grabbing names. But if that's something we're interested in doing, I'm fine with that too.

An alternative would be to see if we can tweak the conventional changelog generator to automate this to look a bit more like https://github.com/ionic-team/stencil-sass/releases, where the author is inlined with the request

Reply:
Let's keep it simple and just do names!